src/daemons/rhsmcertd.c

```diff
@@ -642,6 +734,11 @@ main (int argc, char *argv[])
 
     Config *config = get_config (argc, argv);
 
+    // When --debug CLI option is set, then override configuration option
+    if (show_debug) {
+        log_level = LOG_LEVEL_DEBUG;
+    }
+
```

Comment:
considering that now `key_file_init_config()` does not change the log level if `-d`/`--debug` was specified, the equivalent of this code in `get_config()` does the job just fine, and this bit is redundant now

Reply:
Yeah, good point. It is redundant now.